githubclient.py

```diff
@@ -0,0 +1,20 @@
+from github import Github, GithubException
+
+
+def get_commits_github(reponame, since_date):
+	if not reponame:
+		return []
+
+	g = Github('gofedbot', 'bowie47')
```

Comment:
Yesh :). Can you remove the credentials from the PR (even if it will stop working) and change the password?